$NetBSD: patch-Amaya_amaya_HTMLedit_c,v 1.1.2.2 2012/01/09 04:17:31 sbd Exp $

Hack insane C++ so gcc 4.5 accepts it. No idea if this works. No idea
if it worked before I touched it, either.

--- Amaya/amaya/HTMLedit.c~	2009-12-09 10:11:22.000000000 +0000
+++ Amaya/amaya/HTMLedit.c
@@ -1689,7 +1689,7 @@ void SetREFattribute (Element element, D
                 {
 #ifdef _WX
                   wxString tmpFile = TtaConvMessageToWX(tempURL);
-                  wxFile::wxFile (tmpFile, wxFile::write);
+                  wxFile tmpFileFile (tmpFile, wxFile::write);
 #endif /* _WX */
                 }
             }
